eng/mgmt/automation/generate_utils.py

```diff
@@ -54,12 +54,8 @@ def generate(
     shutil.rmtree(os.path.join(output_dir, 'src/main'), ignore_errors=True)
     shutil.rmtree(os.path.join(output_dir, 'src/test/java', namespace.replace('.', '/'), 'generated'),
                   ignore_errors=True)
-    if os.path.exists(os.path.join(output_dir, 'src/samples/README.md')):
-        # samples contains hand-written code
-        shutil.rmtree(os.path.join(output_dir, 'src/samples/java', namespace.replace('.', '/'), 'generated'),
-                      ignore_errors=True)
-    else:
-        shutil.rmtree(os.path.join(output_dir, 'src/samples'), ignore_errors=True)
+    shutil.rmtree(os.path.join(output_dir, 'src/samples/java', namespace.replace('.', '/'), 'generated'),
+                  ignore_errors=True)
```

Comment:
The repo is cleaned up in recent releases, that all generated samples would be under `<artifact>.generated` namespace. Hence we now don't delete file not there, as they usually be handwritten samples.